Master is not working as nomnom module is not found

Reproduction steps:

```bash
# cd into an empty directory
npm install zaach/jsonlint#master

# Run jsonlint with
node_modules/.bin/jsonlint

# The following error is shown

internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module 'nomnom'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/private/var/folders/24/0j37d7r90zn4pf8bpf5y2ypr0000gn/T/tmp.616RW1vE/node_modules/jsonlint/lib/cli.js:9:15)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
```

--- a/lib/cli.js
+++ b/lib/cli.js
@@ -6,7 +6,7 @@ var parser = require("./jsonlint").parser;
 var JSV = require("JSV").JSV;
 var formatter = require("./formatter.js").formatter;
 
-var options = require("nomnom")
+var options = require("@gerhobbelt/nomnom")
   .script("jsonlint")
   .options({
     file: {

